scheduler/src/cook/mesos/mesos_compute_cluster.clj

```diff
@@ -289,7 +290,22 @@
   (use-cook-executor? [_] true)
 
   (container-defaults [_]
-    container-defaults))
+    container-defaults)
+
+  (retrieve-sandbox-url-path
+    ;"Constructs a URL to query the sandbox directory of the task.
```

Comment:
This commented-out docstring is weird. Can we move it inside the body and wrap it in a `(comment ...)` instead?